cli/damas.sh

```diff
@@ -199,135 +204,146 @@ if [ ! -t 0 -a $# -eq 0 ]; then
 fi
 
 case $COMMAND in
-    init)
-      read -p "remote URL (default = localhost:8090): " URL
-      if [  -z $URL ]; then
-        URL='localhost:8090'
-      fi
-      mkdir '.damas'
-      echo 'URL="http://'$URL'/api/"' > '.damas/config'
-      echo 'Initialized empty Damas repository in '$(realpath .) \
-        '/.damas/ with remote http://'$URL'/api/'
-      exit 0
-      ;;
+  init)
+    read -p "remote URL (default = http://localhost:8090): " URL
+    if [ -z $URL ]; then
+      URL='http://localhost:8090'
+    fi
+    mkdir '.damas'
+    echo 'URL="'$URL'/api/"' > '.damas/config'
+    echo 'Initialized empty Damas repository in '$(realpath .) \
+        '/.damas/ with remote '$URL'/api/'
+    exit 0
+    ;;
 esac
 
 # Verify if in a .damas directory
-upsearch '.damas'
+
+if [ $DAMAS_DIR ]; then
+  DIRECTORY=$DAMAS_DIR
+else
+  upsearch '.damas'
+fi
+
 DIRECTORY=$(realpath $DIRECTORY)
+
 CONFIG=$DIRECTORY'/.damas/config'
 if [ ! -f $CONFIG ]; then
-    echo "config file does not exist. Creating one.."
-    echo 'URL="http://localhost:8090/api/"' > $CONFIG
+  echo "config file does not exist. Creating one.."
+  echo 'URL="http://localhost:8090/api/"' > $CONFIG
```

Comment:
This has not been changed in these commits, but yes, it was only a verification to be sure there is no error when using `source $CONFIG`. It can be changed by anything else